README.md

```diff
@@ -1,2 +1,48 @@
 # hw01
-Repository for homework 01
+
+hw01
+
+## Deena Bernett
+
+Note: I'm auditing this class, so you don't actually have to grade this unless you really want to.
+
+### Personal
+
+
+I'm *old*. I'm married with two kids (ages 5 and 8), and I live far from Hyde Park. Which, makes things fun.
+
+Photo of me illustrating oldness:
+
+![](https://psychology.uchicago.edu/sites/psychology.uchicago.edu/files/styles/columnwidth-wider/public/uploads/images/Bernett.jpg)
```

Comment:
If you want, you can also put the words you'd like to appear in case the picture source is not working, like alternative sources, or "picture missing" in the []